### DISCUSSION TITLE
Using FrankenPHP to build all-in-one binaries

### DISCUSSION THREAD

**Discussion by binaryfire** [laravel-zero/laravel-zero#485] [Ideas]
Hi guys

Have you considered adding FrankenPHP support for creating fully stand-alone binaries? Docs: https://frankenphp.dev/docs/embed/. Perhaps @dunglas has some suggestions...

This would really take things to the next level.

**Comment by osteel** [comment]
Hey there! You might want to check out [this discussion](https://github.com/laravel-zero/laravel-zero/discussions/457) also, which is on a similar topic
